dotnet/samples/Demos/Hosting/Agent/ChatWithAgent.ApiService/ChatWithAgent.ApiService.csproj

```
@@ -15,6 +15,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\ChatWithAgent.ServiceDefaults\ChatWithAgent.ServiceDefaults.csproj" />
+    <ProjectReference Include="..\ChatWithAgent.Configuration\ChatWithAgent.Configuration.csproj" IsAspireProjectResource="false" />
```

Why do we need this to forcibly false?

```suggestion
    <ProjectReference Include="..\ChatWithAgent.Configuration\ChatWithAgent.Configuration.csproj" />
```

---

Yep, otherwise it does not compile:
- https://github.com/dotnet/aspire/issues/2234
- https://learn.microsoft.com/en-us/dotnet/aspire/fundamentals/dotnet-aspire-sdk#project-references